### PR TITLE
Implement Signup controls

### PR DESCRIPTION
**Pull request by SakuraIsayeki**
Signups can now be accepted, controlled or rejected, all within the Viewer. 
Small quirk to note, is that multiple control channels can now be created,

### CODE REVIEW

**Review by MissNathanaelle — APPROVED** [review]
Perfect.

File: Transcom.Web/Startup.cs [hunk 1]
```diff
@@ -37,7 +37,7 @@ public void ConfigureServices(IServiceCollection services)
 			services.AddServerSideBlazor();
 			services.AddHttpContextAccessor();
 
-			if (hostingEnvironment.IsProduction() || true)
+			if (hostingEnvironment.IsProduction())
```
Comment (celi28) [inline Transcom.Web/Startup.cs:40]: Doesn't include case where `type(hostingEnvironment.IsProduction()) = None`.
Be warned of production system without IsProduction prop not defined.

**Review by celi28 — APPROVED** [review]
No issue so far.
Just we warned to edit related configuration for production.

File: Transcom.Web/Startup.cs (hunk 1 above)
Comment (SakuraIsayeki) [reply Transcom.Web/Startup.cs:40]: > Doesn't include case where `type(hostingEnvironment.IsProduction()) = None`.
> Be warned of production system without IsProduction prop not defined.

As we don't use the Staging environment, we're limited to Development and Production environments. These steps are exclusively used on production ^^

**Comment by SakuraIsayeki** [comment]
> No issue so far.
> Just we warned to edit related configuration for production.

Issue has been duly noted. Thank you for your vigilance and review ;)

**Pull request merged**